### PR TITLE
add deploy matic command and related changes

### PR DESCRIPTION
**Pull request by sylar217**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -15,7 +15,8 @@ declare global {
 
 (async function () {
   if (window.ethereum) {
-    window.web3 = new Web3(window.ethereum)
+    window.web3 = new Web3(window.ethereum);
+    await window.ethereum.enable();
```
Comment (shad-k) [inline src/index.tsx:19]: Do we want the user to first connect their wallet, even before they start interacting with the app? I would suggest we ask the user to connect only when uploading memes.

Comment (BTCAlchemist) [reply src/index.tsx:19]: I agree to ask the user only when uploading memes, for now. When we simplify the crypto UX, this would hopefully go away.

Comment (BTCAlchemist) [reply src/index.tsx:19]: @shad-k On the deployed site (https://bafybeibosroaxdm7sa5dyskg4w46jzo5gyaslys7ngvp7wmth4576254jm.on.fleek.co/), the previously uploaded memes do not show up. They show up only when I connect my wallet with Matic testnet active, then refresh the page. What is the reason for this, and may we load the previous memes automatically without wallet testnet authentication?

Comment (matprime) [reply src/index.tsx:19]: I believe the reason for this is that we are currently storing meme with hash of picture on blockchain and meme picture to ipfs. So to read hashes of meme picture we need to read them from blockchain and after that we can fetch pictures from ipfs. After textile integration we will store hashes of pictures in Textile, so I presume for showing memes we will not need access to blockchain and user will not need to connect to wallet to see memes (hashes of meme picture will come from Textile and fetching from picture will be done trough textile). User will need to connect wallet to app only when he will create new meme.

Comment (sylar217) [reply src/index.tsx:19]: Ideal experience would be to connect to wallet only when needed for upload, but right now the data resides in smart contract hence added the same to view the data at home page. Textile integration is pending post which Home page data should load directly from textile.

**Review by matprime — APPROVED** [review]

**Review by shad-k — APPROVED** [review]

**Pull request merged**